src/main/scala/com/wavesplatform/Explorer.scala

```diff
@@ -260,6 +262,51 @@ object Explorer extends ScorexLogging {
           println(txs.length)
           txs.foreach(println)
 
+        case "AP" =>
+          val address   = Address.fromString(args(2)).explicitGet()
+          val portfolio = reader.portfolio(address)
+          log.info(s"$address : ${portfolio.balance} WAVES, ${portfolio.lease}, ${portfolio.assets.size} assets")
+          portfolio.assets.toSeq.sortBy(_._1.toString) foreach {
+            case (assetId, balance) => log.info(s"$assetId : $balance")
+          }
+
+        case "APS" =>
+          val wbals = reader.wavesDistribution(blockchainHeight).explicitGet().toSeq
+          println(s"\nWAVES balances (${wbals.size} addresses)")
+          val addrs = mutable.Set[Address]()
+          wbals.sortBy(_._1.toString).foreach {
+            case (addr, bal) =>
+              println(s"$addr : $bal")
+              addrs += addr
+          }
+
+          val assets = (for {
```

Comment:
Please use `db.iterateOver(Keys.assetInfoHistory(..)`. Also there's an ordering for `ByteStr`, which sorts byte strings lexicographically.